### PR TITLE
Feature/testing

### PR DESCRIPTION
**Pull request by Tchilo**
- Configure redux 
- Add store 
- Fetch data from API 
- Filter data from API 
- Display data on the home page 
- Create links to the details page 
- Add mock file for testing 
- Add testing files 
- Deploy site and add link
- Add a link to video and update README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,43 +65,45 @@ Your app is ready to be deployed!
...
+Feel free to check the [issues page](../../issues/).
 
-### Advanced Configuration
+## Show your support
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+Give a ⭐️ if you like this project!
 
-### Deployment
+## Acknowledgments
```
Comment (Davidosky007) [inline README.md:99]: * Kindly ensure The README contains an "Acknowledgements" section or similar that gives appropriate credit to[ Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License.](https://creativecommons.org/licenses/)

File: src/index.css [hunk 2]
```diff
@@ -1,7 +1,120 @@
...
+.back {
+  color: aqua;
+  font-size: 25px;
+  margin-left: 20px;
+  border-radius: 10px;
+  border: solid aqua 1px;
+}
+
+.heading {
+  margin-bottom: 10px;
```
Comment (Davidosky007) [inline src/index.css:83]: * Kindly follow the design guidelines by using the appropriate fonts family [refer to the design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @Tchilo 👋, 

Good job so far!  💪 
* Nice video presentation ✔️ 
* Well written PR ✔️ 
* Descriptive commit messages ✔️
* No linter errors ✔️ 
* Proper use of git-flow ✔️
* All test passed ✔️   
## There are some issues that you still need to work on to go to the next project but you are almost there!
almost-jumpin-gif



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Tchilo) [reply README.md:99]: oops, my bad I forgot.
I have added the Acknowledgements section... thanks for the review 😄

File: src/index.css (hunk 2 above)
Comment (Tchilo) [reply src/index.css:83]: Added it

**Comment by Davidosky007** [comment]
## Hi @Tchilo 👋, 

## Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 


job-welldone-gif

## Cheers and Happy coding!👏👏👏

## Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**